### PR TITLE
Explain that you can reject BUNDLE. Fixes #159

### LINKED ISSUE #159: Transport stream structure depends on answer to offer when offering bundle.
F. Section 4.1.1:

   Regardless of policy, the
   application will always try to negotiate BUNDLE onto a single
   transport, and will offer a single BUNDLE group across all media
   sections.

I think this sentence may be a bit misleading without additional sentence(s) to explain that the actual number of established transport streams will depend on the peer's response to the attempt to bundle.

### PR DESCRIPTION
**Pull request by ekr**

### CODE REVIEW

**Comment by fluffy** [comment]
LGTM

File: draft-ietf-rtcweb-jsep.xml [hunk 1]
```diff
@@ -793,7 +793,9 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           target="I-D.ietf-mmusic-sdp-bundle-negotiation"></xref>.
           Regardless of policy, the application will always try to negotiate BUNDLE
           onto a single transport,
-          and will offer a single BUNDLE group across all media sections.
+          and will offer a single BUNDLE group across all media sections. The
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: Given the text below that talks about non-BUNDLE-aware endpoints, I think this sentence would be best written as
"... and will offer a single BUNDLE group across all media sections; use of this single transport is contingent upon the answerer accepting BUNDLE."

File: draft-ietf-rtcweb-jsep.xml [hunk 2]
```diff
@@ -793,7 +793,9 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
...
+          actual number of transports negotiated depends on whether the answerer
+          accepts BUNDLE.
           However, by specifying a policy from the list below, the application can
           control how aggressively it will try to BUNDLE media streams together,
           which affects how it will interoperate with a non-BUNDLE-aware endpoint.
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: consider adding "exactly" before "how"

**Comment by juberti** [comment]
Overall looks good - added some minor suggestions

**Comment by juberti** [comment]
lgtm

**Pull request merged**